libraries/tools/kotlin-gradle-plugin/src/test/resources/testProject/kotlinProjectOmitVersion/src/main/kotlin/helloWorld.kt

```diff
@@ -0,0 +1,22 @@
+package demo
+
+import com.google.common.primitives.Ints
+import com.google.common.base.Joiner
+import java.util.ArrayList
+
+class KotlinGreetingJoiner(val greeter : Greeter) {
+
+    val names = ArrayList<String?>()
+
+    fun addName(name : String?): Unit{
+        names.add(name)
+    }
+
+    fun getJoinedGreeting() : String? {
+        val joiner = Joiner.on(" and ").skipNulls();
```

Comment:
why do we need guava in such simple example? we can replace ugly `Joiner.on(" and ").skipNulls()` with `names.filterNotNull().joinToString(" and ")`. We also don't need `com.google.common.primitives.Ints` at all...